plugins/woocommerce/templates/checkout/form-billing.php

```diff
@@ -17,6 +17,12 @@
  */
 
 defined( 'ABSPATH' ) || exit;
+/**
+ * Trigger action before rendering billing fields HTML. Allow you to add custom HTML sections.
+ *
+ * @param checkout instance of WC_Checkout.
```

Comment:
We should add a `@since` tag to the docblock to mark the version it was first added (see https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#4-hooks-actions-and-filters). In this case, the version should be `7.6.0`.

Reply:
`@since` tag added.